docs/language/api.rst

```diff
@@ -436,6 +436,38 @@ Parameters may have the following keywords in front of them:
         => (zig-zag-sum 1 2 3 4 5 6)
         -3
 
+&kwonly
+    *(Python 3 only.)* Parameters that can only be called as
+    keywords. Mandatory keyword-only arguments are declared with the
+    argument's name; optional keyword-only arguments are declared as a
+    two-element list containing the argument name followed by the
+    default value (as with `&optional` above).
+
+    .. code-block:: clj
+
+        => (defn compare [a b &kwonly keyfn [reverse false]]
+        ...  (let [[result (keyfn a b)]]
+        ...    (if (not reverse)
+        ...      result
+        ...      (- result))))
+        => (apply compare ["lisp" "python"]
+        ...        {"keyfn" (fn [x y]
+        ...                   (reduce - (map (fn [s] (ord (first s))) [x y])))})
+        -4
+        => (apply compare ["lisp" "python"]
+        ...        {"keyfn" (fn [x y]
+        ...                   (reduce - (map (fn [s] (ord (first s))) [x y])))
+        ...         "reverse" true})
+        4
+
+    .. code-block:: python
```

Comment:
This was actually intentional; I wanted to show the exception when the mandatory keyword argument was omitted, but Sphinx/Pygments didn't do any syntax highlighting when it was all part of the same Clojure-style block (probably because it got confused by the Python stack trace), whereas if just the part with the stack trace is in a separate Python block, I think it looks OK&mdash;
![riddle_of_the_sphinx_syntax_highlighting](https://cloud.githubusercontent.com/assets/1076988/6661451/cbfe4ce6-cb69-11e4-9121-372c6b5de337.png)